File: packages/scanner/tests/fixtures/VCS-014/vulnerable.js
async function getUser(id) {
  return await db.query(`SELECT * FROM users WHERE id = ${id}`);
}

async function listTable(table) {
  return db.execute("SELECT * FROM " + table);
}

async function search(name) {
  return await db.raw(`SELECT * FROM products WHERE name = '${name}'`);
}

File: packages/scanner/tests/fixtures/VCS-014/vulnerable.py
import sqlite3

def get_user(user_id):
    conn = sqlite3.connect("app.db")
    cursor = conn.cursor()
    cursor.execute(f"SELECT * FROM users WHERE id = {user_id}")
    return cursor.fetchone()

def list_table(table):
    cursor.execute("SELECT * FROM " + table)

def update_status(uid, status):
    cursor.execute("UPDATE users SET status = '%s' WHERE id = %s" % (status, uid))

def search(name):
    cursor.execute("SELECT * FROM products WHERE name = '{}'".format(name))
